tests/js/user-tests.js

```diff
@@ -186,6 +196,23 @@ module.exports = {
       .catch((e) => { Promise.reject(e) } )
   },
 
+  testAuthenticateAdminToken() {
+    if (!isNodeProcess) {
+      return
+    }
+    // read admin token from ROS
+    let obj = JSON.parse(fs.readFileSync('../realm-object-server-data/keys/admin.json', 'utf8'));
```

Comment:
if it's the only usage why not declare it as 
`let obj = JSON.parse(require('fs').readFileSync('../realm-object-server-data/keys/admin.json', 'utf8'));` you can avoid the declaration in https://github.com/realm/realm-js/pull/2038/files#diff-f24a77a955cf2ca9d84fe9d6d07fe08bR33

Reply:
I believe that the RN packager will pick up `fs` anyway when it scans the files.